docs/source/daml/intro/9_Dependencies.rst

```diff
@@ -81,11 +81,11 @@ Structuring Projects
 
 As you've seen here, identifiers depend on the package as a whole and packages always bring all their dependencies with them. Thus changing anything in a complex dependency graph can have significant repercussions. It is therefore advisable to keep dependency graphs simple, and to separate concerns which are likely to change at different rates into separate packages.
 
-For example, in all our projects in this intro, including this chapter, our scripts are in the same project as our templates. In practice, that means changing a test changes all identifiers, which is not desirable. It's better for maintainability to separate tests from main templates. If we had done that in chapter 7, that would also have saved us from copying the chapter 7
+For example, in all our projects in this intro, including this chapter, our scripts are in the same project as our templates. In practice, that means changing a test changes all identifiers, which is not desirable. It's better for maintainability to separate tests from main templates. If we had done that in chapter 7, that would also have saved us from copying the chapter 7.
```

Comment:
Seems like something has gone missing at the end of the last sentence in this paragraph as well. Copying the chapter 7 what?

Reply:
My understanding is that we mean the chapter 7 itself.
@bame-da, if I am not mistaken you are the original author?

Reply:
If so, the best phrasing to use in both instances is "chapter 7" (with no 'the').